packages/backend-data/API.md

```diff
@@ -46,6 +52,7 @@ export type DataProps = {
     authorizationModes?: AuthorizationModes;
     functions?: Record<string, ConstructFactory<AmplifyFunction>>;
     logging?: DataLoggingOptions;
+    migratedAmplifyGen1DynamoDbTableMappings?: AmplifyGen1DynamoDbTableMapping[];

```

Comment:
Do we expect more such props in the future? Should we premptively nest it under `gen1Migration` key to avoid polluting the top level API surface? In the future we can just take away the entire structure easily.

Reply:
I don't expect any more migrations related props in the future. The same concern came up in the API review and it was decided not to nest the property.